app/shared.css

```diff
@@ -151,6 +151,10 @@ Let us know if you think you've found one we'd like. 😏
 	padding: 0;
 }
 
+.page-grid-footer-link.active {
+	color: white;
```

Comment:
A couple of issues here:

* Other colors use [CSS variables](https://developer.mozilla.org/en-US/docs/Web/CSS/Using_CSS_custom_properties) to also support [dark mode](https://developer.chrome.com/docs/devtools/rendering/emulate-css)
* White on our yellow has [not enough color contrast to be accessible](https://webaim.org/resources/contrastchecker/?fcolor=FFFFFF&bcolor=EFDB4F)
    * If you hover your mouse over the element while the Chrome dev tools > Elements tab is open, it'll give you a nice display including the ratio:
    ![Chrome dev tools elements hover showing a 1.4 contrast ratio](https://github.com/philly-js-club/philly-js-club-website/assets/3335181/6b1a321c-0450-4398-9042-d735d5353b24)